codec-memcache/src/main/java/io/netty/handler/codec/memcache/LastMemcacheContent.java

```diff
@@ -90,22 +90,4 @@ public boolean release(int decrement) {
             return false;
         }
     };
-
-    @Override
-    LastMemcacheContent copy();
-
-    @Override
-    LastMemcacheContent retain(int increment);
-
-    @Override
-    LastMemcacheContent retain();
-
-    @Override
-    LastMemcacheContent touch();
-
-    @Override
-    LastMemcacheContent touch(Object hint);
-
-    @Override
-    LastMemcacheContent duplicate();
```

Comment:
same comment as above. This is needed as otherwise it will not return the correct type and so not allow method chaining correctly.